book/src/intro.md

```diff
@@ -15,46 +18,53 @@ _Documentation for the Kona project._
 
 ## Introduction
 
-Kona is _the_ monorepo for the Rust Optimism <a href="https://specs.optimism.io/">OP Stack</a>.
+Kona is _the_ monorepo for the Rust <a href="https://specs.optimism.io/">OP Stack</a>.
 
-Kona provides a suite of libraries and build pipelines for developing verifiable Rust programs targeting
-{{#template ../templates/glossary-link.md root=./ ref=fault-proof-vm text=Fault Proof VMs}}.
+Kona provides a rich suite of libraries and build pipelines. Proof libraries are available for
+developing verifiable Rust programs targeting {{#template ../templates/glossary-link.md root=./ ref=fault-proof-vm text=Fault Proof VMs}}.
+These libraries provide tooling and abstractions around low-level syscalls, memory management,
+and other common structures that authors of verifiable programs will need to interact with.
+It also provides build pipelines for compiling `no_std` Rust programs to a format that can be
+executed by supported Fault Proof VM targets.
 
-It is built and maintained by members of [OP Labs][op-labs] as well as open source contributors,
-and is licensed under the MIT License.
+Protocol libraries are `no_std` compatible for use within the Fault Proof. These types are
+also built for re-use by other components of the OP Stack, for example a rollup node.
 
-Kona provides tooling and abstractions around low-level syscalls, memory management, and other common structures that authors of verifiable programs
-will need to interact with. It also provides build pipelines for compiling `no_std` Rust programs to a format that can be executed by supported
-Fault Proof VM targets.
+Kona is built and maintained by open source contributors and is licensed under the MIT License.
 
 ## Goals of Kona
 
 **1. Composability**
 
-Kona provides a common set of tools and abstractions for developing verifiable Rust programs on top of several supported Fault Proof VM targets. This is done
-to ensure that programs written for one supported FPVM can be easily ported to another supported FPVM, and that the ecosystem of programs built on top of these targets
-can be easily shared and reused.
+Kona provides a common set of tools and abstractions for developing verifiable Rust programs
```

Comment:
A bit out of date as well - needs to be more generic, with kona housing the rollup node etc. this scope is a bit narrow.